#!/usr/bin/env python
PACKAGE = "IP_SAUVC"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("m1min_param",    int_t,    0, "m1min parameter", 0,  0, 260)
gen.add("m1max_param",    int_t,    0, "m1max parameter", 0,  0, 260)
gen.add("m2min_param",    int_t,    0, "m2min parameter", 0,  0, 260)
gen.add("m2max_param",    int_t,    0, "m2max parameter", 0,  0, 260)
gen.add("m3min_param",    int_t,    0, "m3min parameter", 100,  0, 260)
gen.add("m3max_param",    int_t,    0, "m3max parameter", 255,  0, 260)

exit(gen.generate(PACKAGE, "IP_SAUVC", "mat"))
